app/viewmodels/CashAccountDailyStatementsViewModel.scala

```diff
@@ -16,14 +16,19 @@
 
 package viewmodels
 
-import helpers.Formatters.{dateAsDayMonthAndYear, formatCurrencyAmount}
-import models.{CashDailyStatement, CashTransactionType, CashTransactions, Declaration, Payment, Transaction, Transfer, Withdrawal}
+import config.AppConfig
+import helpers.Formatters.{dateAsDayMonthAndYear, formatCurrencyAmount, parseDateString}
+import models.{
+  CashDailyStatement, CashTransactionType, CashTransactions, Declaration, Payment, Transaction,
```

Comment:
this seems oddly formatted to me

Reply:
If you select all the imports and use standard scala formatting or default IntelliJ formatting, it format this way